lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -1008,31 +1007,28 @@ def _init_ticks(self, axis_name, **kw):
         self.minor_ticklabels = TickLabels(size=size, axis=self.axis,
                                            axis_direction=self._axis_direction)
 
-        self.major_ticklabels.set(figure = self.axes.figure,
+        self.major_ticklabels.set(figure=self.axes.figure,
                                   transform=trans,
                                   fontproperties=fontprops)
         self.major_ticklabels.set_pad(major_tick_pad)
 
-        self.minor_ticklabels.set(figure = self.axes.figure,
+        self.minor_ticklabels.set(figure=self.axes.figure,
                                   transform=trans,
                                   fontproperties=fontprops)
         self.minor_ticklabels.set_pad(minor_tick_pad)
 
     def _get_tick_info(self, tick_iter):
         """
-        return ticks_loc_angle, ticklabels_loc_angle_label
-
-        ticks_loc_angle : list of locs and angles for ticks
-        ticklabels_loc_angle_label : list of locs, angles and labels for tickslabels
+        Returns a pair of (list of locs and angles to ticks) and (list of locs,
```

Comment:
I'd go for numpydoc to unwind the *pair of list of and and list of and*:

~~~
Return information on the tics.

Returns
-------
ticks_loc_angle : list of locs and angles for ticks
ticklabels_loc_angle_label : list of locs, angles and labels for tickslabels
~~~

Reply:
The second line doesn't fit in 79 characters so I wanted to avoid an awkward line continuation.  Moreover the return name is basically duplicating the description.

Reply:
What about shortening the `Returns` to
~~~
tick_info : list of locs and angles for ticks
ticklabel_info : list of locs, angles and labels for tickslabels
~~~

Reply:
Here you still have the first 4 lines of the docstring which are basically useless ("Return information about the ticks", blank, "Returns", underline).

For public methods there's (a lot of) value in having things properly formatted for HTML rendering; for private methods I think we should not overdo it.

Reply:
If we're not adhering to the numpydoc conventions, I would basically just leave the original docstring. It's more clear than writing it out in a nested sentence.